BootloaderCommonPkg/Library/SecureBootLib/SecureBootRsa.c

```diff
@@ -47,18 +47,26 @@ DoRsaVerify (
   RETURN_STATUS    Status;
   UINT8           *TmpPubKey;
   RSA_PUB_KEY     *InpPubKey;
+  PUB_KEY         *PublicKey;
   UINT8            Digest[SHA256_DIGEST_SIZE];
   UINT8            PubKeyBuf[RSA_MOD_SIZE + RSA_E_SIZE];
```

Comment:
This assumed RSA2048 already ?

Reply:
As part of RSA3K support, PubKeyBuf would be updated. Currently keeping RSA2K